apps/challenges/serializers.py

```diff
@@ -30,6 +31,22 @@ class Meta:
                   'published', 'enable_forum', 'anonymous_leaderboard', 'is_active',)
 
 
+class ChallengeConfigurationSerializer(serializers.ModelSerializer):
+    """
+    Serialize the Challenge Configuration Model.
```

Comment:
Writing doc string is a good practice. I really appreciate that. Just that it will be great if we can keep the model name as `ChallengeConfiguration` (no spaces).